examples/simple/posix/UB-PTHREAD6-bad.output

```diff
@@ -0,0 +1,7 @@
+Error: UB-PTHREAD6
+Description: Attempting to unlock a PTHREAD_MUTEX_DEFAULT or PTHREAD_MUTEX_NORMAL mutex which is not robust and which has not been locked by the current thread.
```

Comment:
This description is a bit hard to follow.  Can you remove `which is not robust and`?  Since it is an undefined behavior, it is sort of understood that it is not robust.  Or am I misunderstanding it?